src/core/core.js

```diff
@@ -307,9 +307,22 @@ export default class Core {
         this.globalStorage.delete('skipSpellCheck');
         this.globalStorage.delete(StorageKeys.QUERY_TRIGGER);
 
+        const universalSections = data[StorageKeys.UNIVERSAL_RESULTS].sections;
```

Comment:
Do we want the generation of this data in this function? Do we maybe want to split it out into some `getExposedParams`?

Reply:
Split this out into another function!